pennylane/qchem/structure.py

```diff
@@ -435,53 +435,53 @@ def mol_data(identifier, identifier_type="name"):
 
     >>> mol_data("BeH2")
     (['Be', 'H', 'H'],
-     array([[ 4.79405604,  0.29290815,  0.        ],
-            [ 3.77946   , -0.29290815,  0.        ],
-            [ 5.80884105, -0.29290815,  0.        ]]))
+    tensor([[ 4.79404621,  0.29290755,  0.        ],
+            [ 3.77945225, -0.29290755,  0.        ],
+            [ 5.80882913, -0.29290755,  0.        ]], requires_grad=True))
 
     >>> mol_data(223, "CID")
     (['N', 'H', 'H', 'H', 'H'],
-    array([[ 4.79404621,  0.        ,  0.        ],
-            [ 5.80882913,  0.5858151 ,  0.        ],
-            [ 3.77945225, -0.5858151 ,  0.        ],
-            [ 4.20823111,  1.01459396,  0.        ],
-            [ 5.3798613 , -1.01459396,  0.        ]]))
+    tensor([[ 0.        ,  0.        ,  0.        ],
+            [ 1.82264085,  0.52836742,  0.40402345],
+            [ 0.01417295, -1.67429735, -0.98038991],
+            [-0.98927163, -0.22714508,  1.65369933],
+            [-0.84773114,  1.373075  , -1.07733286]], requires_grad=True))
```

Comment:
Nice! I found this same bug, but I found it in the changelog so I only updated it there. Glad its now fixed overall.